apps/api/src/app/notification-template/dto/create-notification-template.request.dto.ts

```diff
@@ -1,7 +1,8 @@
 import { IsArray, IsBoolean, IsDefined, IsOptional, IsString, MaxLength, ValidateNested } from 'class-validator';
-import { ICreateNotificationTemplateDto, IPreferenceChannels } from '@novu/shared';
 import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
-import { MessageFilter } from '../../shared/dtos/message-filter';
+import { ICreateNotificationTemplateDto, IPreferenceChannels } from '@novu/shared';
+
+import { StepFilter } from '../../shared/dtos/step-filter';
```

Comment:
the create and update notification template DTO share the same interface for filters